src/core_plugins/kibana/public/visualize/listing/visualize_landing_page_table.js

```diff
@@ -42,11 +74,13 @@ export class VisualizeLandingPageTable extends React.Component {
     const PAGE_SIZE = 3;
     this.props.fetch(newFilter).then((results) => {
       const items = results.hits;
+      const pagesCount = this.pager.getPagesCount(items.length);
+
       this.setState({
         items,
         isFetchingItems: false,
-        pager: new Pager(items.length, PAGE_SIZE, 1),
-        selectedIds: new SelectedIds()
+        currentPage: Math.min(this.state.currentPage, pagesCount - 1),
```

Comment:
Another question is how the pager should look if there are no items.
Will it show `0 - 0 of 0`?
And if there is 1 item, is it `1 - 1 of 1`?
And for 2, `1 - 2 of 2`